base/operators.jl

```diff
@@ -88,7 +88,10 @@ should therefore also implement [`hash`](@ref).
 Similar to [`==`](@ref), except for the treatment of floating point numbers
 and of missing values. `isequal` treats all floating-point `NaN` values as equal
 to each other, treats `-0.0` as unequal to `0.0`, and [`missing`](@ref) as equal
-to `missing`. Always returns a `Bool` value.
+to `missing`. Always returns a `Bool` value. It can be assumed that two values which
+are `===` are also `isequal`. `isequal` is an equivalance relation - it is reflexive
+(`isequal(a, b)` implies `isequal(a, b)`) and transitive (`isequal(a, b)` and
```

Comment:
I understand reflexive to mean `isequal(a,a)` implies `true`.

Reply:
The corresponding property here would be "symmetric" instead

Reply:
Duh, thanks